package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@revenuecat/purchases-js",
   "description": "Web subscriptions made easy. Powered by RevenueCat",
   "private": false,
-  "version": "0.18.0",
+  "version": "0.18.1",
```

Comment:
NABD but this is outdated since we're now in 0.18.2 (probably outdated in the base branch)

Reply:
Thanks, we'll need to bump to 1.0.0 on fon/new-sdk-views, I'll roll it back to `0.18.0` and then we can bump in the base branch